### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Lextuga007**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -24,41 +24,51 @@ knitr::opts_chunk$set(
...
 
-## Datasources for reuse
+This package has been created to help NHS, Public Health and related analysts/data scientists learn to use `R`. 
+It contains several free datasets with help files explaining their structure, and `vignette` examples of their use.  
+
+We encourage contributions to the package, both to expand the set of training material, and also as development for newer `R`/github users as a first or early contribution.  
+Please add relevant free, open source data sets that you think may benefit the NHS-R Community.
 
 This package has been created to help NHS, Public Health and related analysts/data scientists learn to use `R`. It contains several free datasets (just one at the moment), help files explaining their structure, and `vignette` examples of their use.  We encourage contributions to the package, both to expand the set of training material, but also as development for newer `R`/github users as a first contribution.  Please add relevant free, open source data sets that you think may benefit the NHS-R Community.
+>>>>>>> 87800e0b30f9343e0d00493ca9daf6040c72790e
```
Comment (tomjemmett) [inline README.Rmd:36]: should this line be deleted?

File: README.Rmd [hunk 2]
```diff
@@ -24,41 +24,51 @@ knitr::opts_chunk$set(
...
 
 ## Installation instructions
 
-This packages is available on CRAN or the development version can be installed from source, via this Github repository.  You will need [`Rtools`](https://cran.r-project.org/bin/windows/Rtools/) installed to build the package, and the `remotes` package.
+You can install the [CRAN](https://CRAN.R-project.org) with:
```
Comment (tomjemmett) [inline README.Rmd:40]: should this be

> You can install the package from CRAN

File: README.Rmd [hunk 3]
```diff
@@ -24,41 +24,51 @@ knitr::opts_chunk$set(
...
 
-```{r install, eval=FALSE}
-remotes::install_github("https://github.com/nhs-r-community/NHSRdatasets")
+``` r
+install.packages("NHSRdatasets")
 ```
 
-## Contribution
+To install the development version from [GitHub](https://github.com/) use:
```
Comment (tomjemmett) [inline README.Rmd:46]: change use to with

**Review by tomjemmett — CHANGES_REQUESTED** [review]

**Review by tomjemmett — CHANGES_REQUESTED** [review]
spotted what looks like a duplicated line, but that may be because I'm viewing on mobile. minor pedantic point, but looks like previously we used "packages" and have continued that, when it should be "package". the point about how to cite the package could become a new issue to solve later

File: README.Rmd [hunk 4]
```diff
@@ -24,41 +24,50 @@ knitr::opts_chunk$set(
...
-* Where based on patient data, the contributor takes full responsibility for sharing the data and certifies that. it is has been processed, anonymised, aggregated or otherwise protected in accordance with all legal requirements under General Data Protection Regulation (GDPR), or other relevant legislation.
+* The dataset does not contain real patient-level or any other sensitive data.
+* Where it is based on sensitive or patient data, the contributor takes full responsibility for sharing the data and certifies that it has been processed, anonymised, aggregated or otherwise protected in accordance with all legal requirements under General Data Protection Regulation (GDPR), or other relevant legislation.
 
+# Citation
 
-Please note that the 'NHSRdatasets' project is released with a
-[Contributor Code of Conduct](https://github.com/nhs-r-community/NHSRdatasets/blob/master/CODE_OF_CONDUCT.md).
-By contributing to this project, you agree to abide by its terms.
+Please cite the use of {NHSRdatasets} and promote the work of the NHS-R
```
[the remark this answers is not in the document]
Comment (Lextuga007) [reply README.Rmd:67]: Good question, I'll look into it.

[the remark this answers is not in the document]
Comment (Lextuga007) [reply README.Rmd:67]: https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/ gives details. I'm not sure it's necessary, we don't cite the use of many packages but I'll leave the request to promote as that's kind of what we want 😄

**Review by tomjemmett — COMMENTED** [review]
Has readme been re-rendered?

**Pull request merged**